### ISSUE TITLE
Bug when running structural reforms which include a `neutralize_variable` command 

### ISSUE THREAD

**Issue by PavelMakarchuk** [PolicyEngine/policyengine-app#1940]
As an example:
[This](https://policyengine.org/us/household?focus=householdOutput.netIncome&household=45963) household in the baseline receives the New York CTC of $1,650
The NY WFTC reform includes a provision to remove the NY CTC
Using the same household, in [this](https://policyengine.org/us/household?focus=householdOutput.netIncome&reform=62046&baseline=2&timePeriod=2024&region=ny&household=45987) case, the NY CTC is not displayed as being received in the baseline scenario and no difference is shown
Local tests and the python package display the change properly which leads me to this that it is an APP issue

**Comment by anth-volk** [comment]
Thanks for filing this. As a thought, this could also be the API.

**Comment by anth-volk** [comment]
I just ran this locally, and want to be sure. The only issue I see from the household at https://policyengine.org/us/household?focus=householdOutput.netIncome&reform=62046&baseline=2&timePeriod=2024&region=ny&household=45987 is that there's nothing to indicate that the WFTC is a part of the increase in credits, but I'm not seeing the issue you had originally indicated.

<img width="1440" alt="Screen Shot 2024-07-30 at 9 22 37 PM" src="https://github.com/user-attachments/assets/00b4e810-d465-4fab-816e-5c2c682a476f">

**Comment by anth-volk** [comment]
This is likely very similar to what's going on in #1942. Basically, the metadata that defines the variable tree for the front end isn't updated to reflect structural reforms' variable tree changes, causing the app to not properly display credits amended by the structural reform. This will likely require some sort of API change.

**Comment by nikhilwoodruff** [comment]
Yeah this seems tricky to solve- I think https://github.com/PolicyEngine/policyengine-app/issues/1949 will supersede

**Comment by nikhilwoodruff** [comment]
Will aim to get it in by tomorrow

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
